source/ways-of-working/guides/index.html.md.erb

```diff
@@ -7,4 +7,5 @@ weight: 98
 
 # <%= current_page.data.title %>
 
-- [Accessing apps via the VPN](connect-via-vpn.html)
\ No newline at end of file
+- [Accessing apps via the VPN](connect-via-vpn.html)
+- [Environments](environments.html)
```

Comment:
can we move this to `Concepts and standards`

Reply:
moved